false
{"a": [2, [3, null]]}
Exception
Exception
